app/styles/app.scss

```diff
@@ -210,3 +224,8 @@ li.active > a {
 .column-offset {
   height: 120px;
 }
+
+.disabled {
+  pointer-events: none;
```

Comment:
Thoughts on switching our stylesheets to 4-space indents, for consistency?

Reply:
Works for me 👍